extensions/clip-swap/src/swap.ts

```diff
@@ -0,0 +1,21 @@
+import { Clipboard, getSelectedText } from "@raycast/api";
+
+export default async function Swap() {
+  const selectedText = await getSelectedText();
+  const clipboardContent = await Clipboard.readText();
```

Comment:
You might need to handle errors from them:

![](https://github.com/user-attachments/assets/5ef38795-aa5b-49c1-bf6e-a2574385e7c9)

The `getSelectedText()` can throw an error when it is unable to get selected text.

Reply:
I've updated this with error handling and opted to just have an error be a no-op as that's the typical behaviour you see with OS-level copy/paste.